sdks/python/apache_beam/io/datastore/v1/datastoreio_test.py

```diff
@@ -118,6 +121,45 @@ def test_SplitQueryFn_with_query_limit(self):
       self.assertEqual(1, len(returned_split_queries))
       self.assertEqual(0, len(self._mock_datastore.method_calls))
 
+  def test_DatastoreWriteFn_with_emtpy_batch(self):
+    self.check_DatastoreWriteFn(0)
+
+  def test_DatastoreWriteFn_with_one_batch(self):
```

Comment:
Use ' _WRITE_BATCH_SIZE' in these tests. In case the constant will change in the other class.